$NetBSD: patch-base64.cabal,v 1.1 2023/01/30 07:59:38 wiz Exp $

Unrestrict 'base' dependency, HASKELL_UNRESTRICT_DEPENDENCIES does
too much since 'base' matches 'base64'.

--- base64.cabal.orig	2023-01-30 07:53:35.450363448 +0000
+++ base64.cabal
@@ -51,7 +51,7 @@ library
     Data.ByteString.Base64.Internal.W64.Loop
 
   build-depends:
-      base           >=4.14     && <4.17
+      base           >=4.14
     , bytestring     >=0.10     && <0.12
     , deepseq        >=1.4.3.0  && <1.5
     , text
@@ -68,7 +68,7 @@ test-suite tasty
   other-modules:    Internal
   main-is:          Main.hs
   build-depends:
-      base               >=4.14 && <4.17
+      base               >=4.14
     , base64
     , base64-bytestring
     , bytestring
@@ -88,7 +88,7 @@ benchmark bench
   hs-source-dirs:   benchmarks
   main-is:          Base64Bench.hs
   build-depends:
-      base               >=4.14 && <4.17
+      base               >=4.14
     , base64
     , base64-bytestring
     , bytestring
